internal/olm/fbcutil/util_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fbcutil
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("test fbutil", func() {
+
+	emptyListHash := fmt.Sprintf("%x", sha256.New().Sum(nil))
+
+	Context("test dirNameFromRefs", func() {
+		DescribeTable("should name the directory with sha256 of the concatenation of the bundle image names", func(refs []string, expectedHash string) {
+			dirName := dirNameFromRefs(refs)
+
+			Expect(dirName).Should(HaveLen(sha256.Size * 2)) // in hex representation, each byte is two hex digits
+			Expect(dirName).Should(Equal(expectedHash))
+		},
+			Entry("long image names", []string{
+				"registry.build03.ci.openshift.org/ci/op/dl9pitlh/pipeline@sha256:123d009f7fb9d5b760b3eac831e0f50413d36199816d651b569367642c138ccd",
+				"registry.build03.ci.openshift.org/ci/op/dl9pitlh/pipeline@sha256:5498f1b841946c8da823fb11c08969efb36119f7867701f2cedac4a2e8f7cefc",
```

Comment:
Might be nice to use non-OpenShift references here. I imagine we can use any long string here